src/PresentationalComponents/TableToolbar/TableToolbar.scss

```diff
@@ -2,7 +2,7 @@
 
 :root {
     --ins-c-table-toolbar--PaddingVertical: 16px;
-    --ins-c-table-toolbar--PaddingHorizontal: 32px;
+    --ins-c-table-toolbar--PaddingHorizontal: 0px;
```

Comment:
This statement is needed for the toolbar to work as intended. If there is other padding, it is coming from somewhere else. 
![Screen Shot 2019-03-13 at 3 50 25 PM](https://user-images.githubusercontent.com/12520842/54309992-be88c480-45a7-11e9-99c5-fed5d9ac9b3b.png)

Reply:
Show me this example in relation to the table. This issue was raised because the toolbar and table left/right edges weren't aligned. Perhaps instead it'd be better to add padding to the table?

Reply:
Sure thing:

![Screen Shot 2019-03-14 at 8 33 22 AM](https://user-images.githubusercontent.com/12520842/54357179-dc9c0680-4633-11e9-810d-1c15de68505e.png)

Reply:
If you're doing:
```
<TableToolbar>
    <Toolbar>
.
.
.
```
That is incorrect. The `TableToolbar` takes place of the `Toolbar` component. https://github.com/RedHatInsights/insights-frontend-components/blob/master/doc/components/tableToolbar.md

Reply:
Ok, thanks!